day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,21 @@
+frogs = Array.new (12) {|x| x - 1}
+frogs.reverse.each do |frog|
+  if frog > 1
+    puts "#{frog} speckled frogs sat on a log"
+    puts "eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
+    puts "then there were #{frog - 1} speckled frogs."
+  elsif frog == 1
+    puts "#{frog} speckled frogs sat on a log"
+    puts "eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
+    puts "then there were no more speckled frogs."
+  else frog >= 1
```

Comment:
This empty else is an anti-pattern. In general the else statement covers the rest of possible outcomes.